src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -331,6 +351,16 @@ public void startTransportService(TransportService transportService) {
             ((request, channel, task) -> channel.sendResponse(updateSettingsRequestHandler.handleUpdateSettingsRequest(request)))
         );
 
+        logger.info(jobDetails.toString());
```

Comment:
This log is unnecessary to add at this point, since we're merely registering the request handler. I would suggest that we utilize logging whenever we send a request, or receive a response. Adding a log here would not add much information to those that review the log.

Reply:
ACK